### PR TITLE
KAA-1621: Nix guide - reviewed. [master]

### PR DESCRIPTION
**Pull request by hunch7**

### CODE REVIEW

File: doc/Customization-guide/Nix-guide/index.md [hunk 1]
```diff
@@ -11,65 +11,68 @@ sort_idx: 70
 {% include variables.md %}
 
 [Nix](http://nixos.org/nix/) is a powerful Linux package manager.
-We use it to create a better development environment for the Kaa C SDK and manage all third-party dependencies.
+Kaa team uses it to improve development environment for the [Kaa C SDK]({{root_url}}Programming-guide/Using-Kaa-endpoint-SDKs/C/) and to manage all third-party dependencies.
```
Comment (rasendubi) [inline doc/Customization-guide/Nix-guide/index.md]: We also use Nix for C++ SDK and documentation generator.

File: doc/Customization-guide/Nix-guide/index.md [hunk 2]
```diff
@@ -11,65 +11,68 @@ sort_idx: 70
...
 
-Note that each time you enter shell you should execute the following command.
+The first time you enter the shell environment, Nix will install all the dependencies required for development.
+
+Each time you enter shell you should execute the following command.
 
 ```sh
 nix-shell
 ```
+{:.note}
```
Comment (rasendubi) [inline doc/Customization-guide/Nix-guide/index.md]: I'm not sure why this is a note?

File: doc/Customization-guide/Nix-guide/index.md [hunk 3]
```diff
@@ -11,65 +11,68 @@ sort_idx: 70
...
-Follow the instructions provided on the screen.
-Then re-run the command above.
+Since CC3200 SDK is not freely available, `nix-shell` will stop the first time you enter and ask you to download the SDK file manually and add it to `nix-store`.
+Follow the on-screen instructions, then run the above command again.
 
-# Using shell
-After dependencies are installed, you'll find yourself in a custom bash shell (you can enter it with `nix-shell` whenever you want).
+## Use shell
+After you installed the dependencies, you will be switched to the custom bash shell.
+You can enter it any time it using `nix-shell`.
```
Comment (rasendubi) [inline doc/Customization-guide/Nix-guide/index.md]: > it any time it

File: doc/Customization-guide/Nix-guide/index.md [hunk 4]
```diff
@@ -11,65 +11,68 @@ sort_idx: 70
...
 
-You can use all your development tools, such as `./build.sh` and `cmake`, from there.
-Furthermore, there is a custom top-level Makefile, provided by default, that propagates your commands to all targets (it also configures all targets appropriately).
-So just run `make` to build the Kaa C SDK for all platforms.
+From there, you can use all your development tools, such as `./build.sh` and `cmake`.
+A custom top-level Makefile, provided by default, propagates your commands to all targets.
+It also configures all targets appropriately.
+To build the Kaa C SDK for any supported platform, run `make` .
```
Comment (rasendubi) [inline doc/Customization-guide/Nix-guide/index.md]: Remove space before dot.

File: doc/Customization-guide/Nix-guide/index.md (hunk 1 above)
Comment (hunch7) [reply doc/Customization-guide/Nix-guide/index.md]: Fixed.

File: doc/Customization-guide/Nix-guide/index.md (hunk 2 above)
Comment (hunch7) [reply doc/Customization-guide/Nix-guide/index.md]: Fixed.

File: doc/Customization-guide/Nix-guide/index.md (hunk 3 above)
Comment (hunch7) [reply doc/Customization-guide/Nix-guide/index.md]: Fixed.

File: doc/Customization-guide/Nix-guide/index.md (hunk 4 above)
Comment (hunch7) [reply doc/Customization-guide/Nix-guide/index.md]: Fixed.

File: doc/Customization-guide/Nix-guide/index.md [hunk 5]
```diff
@@ -11,7 +11,7 @@ sort_idx: 70
 {% include variables.md %}
 
 [Nix](http://nixos.org/nix/) is a powerful Linux package manager.
-Kaa team uses it to improve development environment for the [Kaa C SDK]({{root_url}}Programming-guide/Using-Kaa-endpoint-SDKs/C/) and to manage all third-party dependencies.
+Kaa team uses it to improve development environment for Kaa [C]({{root_url}}Programming-guide/Using-Kaa-endpoint-SDKs/C/)/[C++]({{root_url}}Programming-guide/Using-Kaa-endpoint-SDKs/C++/) SDKs, and to manage all third-party dependencies including the ones required for Kaa document generator. <!--TODO: link to Jekver repository when ready-->
```
Comment (rasendubi) [inline doc/Customization-guide/Nix-guide/index.md]: document -> documentation

Comment (hunch7) [reply doc/Customization-guide/Nix-guide/index.md]: Fixed.

**Review by rasendubi — APPROVED** [review]

**Pull request merged**